yarn-project/simulator/src/client/simulator.test.ts

```diff
@@ -59,35 +56,6 @@ describe('Simulator', () => {
     const createNote = (amount = 123n) =>
       new Note([new Fr(amount), new Fr(0), ownerMasterNullifierPublicKey.hash(), Fr.random()]);
 
-    it('should compute note hashes and nullifier', async () => {
```

Comment:
I decided to nuke this test because it relied on having access to the generators to get the same note hash. This is brittle as we needed to manually copy them and I don't think it provides value as we never need to compute the note hash in typescript (out of tests) and instead always is note specific. So keeping the TS and Noir code in match just seems like an unnecessary hassle.